AWSClientRuntime/Sources/Middlewares/RetryerMiddleware.swift

```diff
@@ -7,6 +7,7 @@
 
 import ClientRuntime
 
+// swiftlint:disable force_cast
```

Comment:
if we are going to use this, we should scope it to the function it's being used in.

Reply:
will do